01-exercise-easy/07-point-counter/pointCounter_Dev.py

```diff
@@ -0,0 +1,14 @@
+def calcular_pontuacao(moedas):
+    cascalho = 0
```

Comment:
Novamente não ficou claro com o que foi solicitado na task o nome de sua variável.
```suggestion
    total_pontos = 0
```